<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>500</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QTableWidget" name="tableWidget">
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
      <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
       <bool>true</bool>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <row>
       <property name="text">
        <string>1</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>2</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>3</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>4</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>5</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>6</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>7</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>8</string>
       </property>
      </row>
      <column>
       <property name="text">
        <string>Artist</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Title</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Album</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Genre</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Comment</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Track</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Year</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string notr="true">Audio file path</string>
       </property>
       <property name="toolTip">
        <string notr="true"/>
       </property>
       <property name="whatsThis">
        <string notr="true"/>
       </property>
      </column>
      <column>
       <property name="text">
        <string>ID</string>
       </property>
      </column>
      <item row="0" column="0">
       <property name="text">
        <string>abc</string>
       </property>
      </item>
      <item row="0" column="1">
       <property name="text">
        <string>zad</string>
       </property>
      </item>
      <item row="0" column="8">
       <property name="text">
        <string>1</string>
       </property>
      </item>
      <item row="1" column="0">
       <property name="text">
        <string>cdf</string>
       </property>
      </item>
      <item row="1" column="1">
       <property name="text">
        <string>gd</string>
       </property>
      </item>
      <item row="1" column="8">
       <property name="text">
        <string>2</string>
       </property>
      </item>
      <item row="2" column="0">
       <property name="text">
        <string>bdf</string>
       </property>
      </item>
      <item row="2" column="1">
       <property name="text">
        <string>sde</string>
       </property>
      </item>
      <item row="2" column="8">
       <property name="text">
        <string>3</string>
       </property>
      </item>
      <item row="3" column="8">
       <property name="text">
        <string>4</string>
       </property>
      </item>
      <item row="4" column="8">
       <property name="text">
        <string>5</string>
       </property>
      </item>
      <item row="5" column="8">
       <property name="text">
        <string>6</string>
       </property>
      </item>
      <item row="6" column="8">
       <property name="text">
        <string>7</string>
       </property>
      </item>
      <item row="7" column="8">
       <property name="text">
        <string>8</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QTextBrowser" name="notFound">
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'.SF NS Text'; font-size:13pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:36pt; color:#f30000;&quot;&gt;NOT FOUND!!&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="separator"/>
   <addaction name="actionExit"/>
   <addaction name="actionChange_folder"/>
   <addaction name="action_Save"/>
   <addaction name="action_Cancel"/>
   <addaction name="actionUpdate"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionChange_folder"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menu_Edit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <addaction name="action_Save"/>
    <addaction name="action_Cancel"/>
   </widget>
   <widget class="QMenu" name="menu_Update">
    <property name="title">
     <string>&amp;Update</string>
    </property>
    <addaction name="actionUpdate"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Edit"/>
   <addaction name="menu_Update"/>
  </widget>
  <action name="actionMenu">
   <property name="icon">
    <iconset>
     <normaloff>:/rec/Open-file-icon.png</normaloff>:/rec/Open-file-icon.png</iconset>
   </property>
   <property name="text">
    <string>Menu</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionChange_folder">
   <property name="text">
    <string>Change folder</string>
   </property>
  </action>
  <action name="action_Save">
   <property name="text">
    <string>&amp;Save</string>
   </property>
  </action>
  <action name="action_Cancel">
   <property name="text">
    <string>&amp;Cancel</string>
   </property>
  </action>
  <action name="actionUpdate">
   <property name="text">
    <string>Update</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
